src/Core/Enums/RuleId.cs

```diff
@@ -128,9 +121,6 @@ public enum RuleId
         // Axe.Windows.Rules
         EditSupportsIncorrectRangeValuePattern,
 
-        [Obsolete("This rule has been deprecated", true)]
-        EditSupportsOnlyValuePattern,
```

Comment:
Is it expected that this rule ID still has corresponding data in: `HowToFix.resx`, `HowToFix.Designer.cs`, `Descriptions.resx`, and `Descriptions.Designer.cs`?

Reply:
These must have been missed when the rule was removed 😦 

Now I have to wonder how many other "orphans" exist...